.github/workflows/Release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release CI
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "Project.toml"
+jobs:
+  branch:
+    name: "Create Release Branch"
+    runs-on: ubuntu-20.04-16core
+    strategy:
+      fail-fast: false
+    steps:
+        - uses: actions/checkout@v3
+        - uses: julia-actions/setup-julia@v1
+          with:
+            version: "1.8"
+            arch: "x64"
+        - uses: julia-actions/cache@v1
+        - name: Check Project version
+          id: version_check
+          shell: julia --color=yes --project {0}
+          run: |
+            using Pkg.Types
+            project = read_project("Project.toml")
+            latest_tag = readchomp(`git describe --tags --abbrev=0`)
+            @assert project.version == latest_tag "$(project.version) != $latest_tag"
+            open(ENV["GITHUB_OUTPUT"], "a") do io
+              println(io, "version=$(project.version)")
+            end
+          continue-on-error: true
```

Comment:
I want to "allow_failure" here and not have the CI fail just because the version and tag are the same

Reply:
Why not assert that `project.version >= latest_tag`?

Reply:
becuase we don't want the rest of the worflow to proceeed if `project.version == latest_tag`

I want this step to fail (but the pipeline to "pass") if `project.version == latest_tag`.
And I only want the rest of the workflow to run if `project.version > latest_tag`

Reply:
I've restructured the workflow based on the point below of wanting it to run (as much as possible) on PRs